docs/product-and-design/copy-delivery.md

```diff
@@ -10,34 +10,49 @@ Translation strings include all application copy, including:
 - In-line link URLs
 - Error messages (like [no script](https://github.com/cal-itp/benefits/blob/main/benefits/core/templates/core/includes/noscript.html), [no cookies](https://github.com/cal-itp/benefits/blob/main/benefits/core/templates/core/includes/nocookies.html) warnings)
 
-## Cal-ITP Benefits Application Copy
+Developers create the 'django.po' message files from copy created and implemented by Design and Product. Design and Product use the tools Figma, Ditto, and Crowdin to craft product copy, manage all copy and variantions of copy (e.g. agency-specific copy or Spanish translation copy), and translate copy.
 
-The human-readable version of the English and Spanish translation strings for the application are delivered to Design and Engineering by Product, and live at this link: [Cal-ITP Benefits Application Copy](https://docs.google.com/spreadsheets/d/1_Gi_YbJr4ZuXCOsnOWaewvHqUO1nC1nKqiVDHvw0118/edit#gid=0).
+## Tools used in Cal-ITP Benefits Application Copy
 
-By tabs:
+### Figma
 
-- `EN-USA` tab contains all copy for English, which each row representing a page. This copy uses a sample agency, called California State Transit (CST) with an Agency Card. This copy is used in Figma.
-- `forTranslation` and `All Agencies` tab contains the English and Spanish translation side by side, with agency-specific copy.
+Figma is our primary design tool and is the source of truth for all design decisions, concepts, and directions for the Benefits app and all Cal-ITP sites. We have Figma Organization which includes developer mode, version history, and branching–a tool to explore new design directions.
+
+The development team prefers not to have admin or editor capabilities so as not to accidentally interfere with any designs. Currently there is no formal process within Figma for handoff. Instead handoffs occur over the course of two meetings, a review of the designs and then what we call “issue-fest,” or a chance for developers to create Github issues while designers can weigh in and answer questions.
```

Comment:
```suggestion
The development team prefers not to have admin or editor capabilities so as not to accidentally interfere with any designs. Currently there is no formal process within Figma for handoff. Instead handoffs occur over the course of two meetings, a review of the designs and then what we call “issue-fest,” or a chance for developers to create GitHub issues while designers can weigh in and answer questions.
```

@srhhnry Aren't we gonna use the Ready for dev feature in Figma for handoff? 
<img width="280" alt="image" src="https://github.com/user-attachments/assets/1ef5437e-7152-461b-ae73-373072eda594">

- If anyone (Product, Engineering) has concerns about copy, should they put a comment on Ditto, on Figma, or some other way?

Reply:
If your preference is for comments in Figma, then we can make that the policy. You can note that all comments on copy or design should be made in Figma, and that all comments need to be submitted by the Hand-Off meeting. No comments should be added after the Hand-Off meeting ( as in, engineers cannot ask design to change anything _after_ hand-off. that would be a new design process ).